packages/slice-machine/package.json

```diff
@@ -31,6 +33,8 @@
     "@prismicio/mocks": "2.0.0-alpha.2",
     "@prismicio/types-internal": "2.0.0-alpha.9",
     "@slicemachine/manager": "0.2.2-dev-page-types.1",
+    "@vanilla-extract/css": "1.11.0",
```

Comment:
❓ Do you have a preference between "1.11.0" and "^1.11.0"?
On my side, I always use the "1.11.0" form because I like to know the exact version directly from package.json

Reply:
Nice, I like this also then!